app/assets/javascripts/covid19/src/data/Covid19InfoFromJsonRepository.ts

```diff
@@ -248,8 +251,16 @@ function extractField(structure: Structure, field: Field): string {
             return getFields(structure.organisms, ["id", "name", "commonName"]);
         case "ligands":
             return getFields(structure.ligands, ["id", "name", "details"]);
+        case "details":
+            return "";
```

Comment:
So the details field is not to be used in the search, right?

Reply:
It should be used in the search, yes.